tools-local/tasks/mobile.tasks/WasmBundleTask/TimeZoneLoader.cs

```diff
@@ -0,0 +1,96 @@
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Text;
+using System.Text.Json;
+using System.Text.Json.Serialization;
+using System.Linq;
+using System.Text.RegularExpressions;
+using System.Net;
+using System.Reflection;
+using System.Diagnostics;
+using Microsoft.Build.Framework;
+using Microsoft.Build.Utilities;
+
+public class TimeZoneLoader : Task
```

Comment:
It seems like the other build tasks are named liked `WasmAppBuilder` or `PInvokeTableGenerator`

Reply:
The mobile.tasks do seem to be named like that, but not others like in `installer.tasks`. IMHO, the task names should read like actions, that get executed in a target. Not a blocker though.

Reply:
Since this is a new task that we are creating, we can follow the msbuild conventions. The idea here is that a msbuild Target executes a bunch of Tasks, which are actions/code, so, you would name the Tasks like a method.
Let's change the names for the new tasks to match that.